kernelci/config/build.py

```diff
@@ -304,14 +331,19 @@ def __init__(self, name, tree, branch, variants):
 
         *variants* is a list of BuildVariant objects, to define all the
                    variants to build for this tree / branch combination.
+
+        *reference* is a Reference object which defines the tree and branch
+                    for bisections.
         """
         self._name = name
         self._tree = tree
         self._branch = branch
         self._variants = variants
+        self._reference = reference
 
     @classmethod
-    def from_yaml(cls, config, name, trees, fragments, build_envs, defaults):
+    def from_yaml(cls, config, name, trees, fragments, build_envs, defaults,
```

Comment:
I actually think the reference defaults should be part of the `defaults` - see my other comment in build-configs.yaml